R/call_waterinfo.R

```diff
@@ -70,14 +70,16 @@ call_waterinfo <- function(query, base_url = "vmm", token = NULL) {
         regexpr(pattern, custom_error,
                 perl = TRUE
         ))
-    }
-    if (grep(pattern = "Credit limit exceeded", error_message) == 1) {
-      error_message <- paste(error_message,
-          "- When you require more extended data requests, please request",
-          "a download token from the waterinfo.be site administrators via",
-          "the e-mail address hydrometrie@waterinfo.be with a statement of",
-          "which data and how frequently you would like to download data.",
-          "Run `?wateRinfo::token` for more information on token usage.")
+      if (isTRUE(grep(pattern = "Credit limit exceeded", error_message))) {
+        error_message <- paste(error_message,
+            "- When you require more extended data requests, please request ",
```

Comment:
@stijnvanhoey `paste()` uses space as the default separator, so no need to add spaces at the end of every lines (it results in double spaces).

Reply:
Fixed it.